manage_app.py

```diff
@@ -236,35 +333,70 @@ def check_service_up(url: str, service: str, discord=False):
     """
 
     print()
-    print(bcolors.YELLOW_IN +
-          "CHECKING SERVICE {}:{} ".format(ENV, service) + bcolors.CEND)
+    print(
+        bcolors.YELLOW_IN +
+        "CHECKING SERVICE {}:{} ".format(
+            os.getenv("DOCK_ENV"), service,
+        ) + bcolors.CEND,
+    )
 
     req_method = ""
 
     if service == "WEB":
         try:
-            web_request = requests.get(url)
+            web_request = requests.get(
+                url, verify=False if ENV != "PROD" else True,
+            )
             req_method = "GET"
 
         except Exception as e:
             return handle_exception(msg, req_method, url, service, e, discord)
 
         if web_request.status_code != 200:
             return handle_status_code_exception(
-                msg, req_method, url, service, web_request.status_code, discord)
+                msg, req_method, url, service, web_request.status_code, discord,
+            )
 
     elif service == "API":
-
         try:
-            web_request = requests.post(url=url)
+            web_request = requests.post(
+                url=url, verify=False if ENV != "PROD" else True,
+            )
             req_method = "POST"
 
         except Exception as e:
             return handle_exception(msg, req_method, url, service, e, discord)
-
         if web_request.status_code != 400:
             return handle_status_code_exception(
-                msg, req_method, url, service, web_request.status_code, discord)
+                msg, req_method, url, service, web_request.status_code, discord,
+            )
+
+    elif service == "PY-API":
+        try:
+            web_request = requests.get(
+                url=url, verify=False if ENV != "PROD" else True,
+            )
+
+        except Exception as e:
+            return handle_exception(msg, req_method, url, service, e, discord)
+
+        if web_request.status_code != 200:
+            return handle_status_code_exception(
+                msg, req_method, url, service, web_request.status_code, discord,
+            )
+
+    elif service == "URL-SHORTENER":
+        try:
+            web_request = requests.get(
+                url=url, verify=False if ENV != "PROD" else True,
```

Comment:
Also, whenever you have such repetitive ternary, you can extract it as a variable

SHOULD_VERIFY=...